### ISSUE TITLE
Port improvements to upstream :smiley:

### ISSUE THREAD

**Issue by goyalyashpal** [trol73/mucommander#193]
Hi! 
Thanks for adding features to muCommander and keeping those FOSS.

Now that the upstream project is maintained again at [mucommander/mucommander](https://github.com/mucommander/mucommander), would u consider making pull requests up there?

Thanks again for improvements 😃 

----
> Unfortunately the muCommander project was abandoned by the autors and not supported since 2012. It lacks such basic and necessary features, such as searching for files. After several unsuccessful attempts to contact the authors, I decided to create the clone muCommander
\- [https://trolsoft.ru/en/soft/trolcommander](https://web.archive.org/web/20221203131435/https://trolsoft.ru/en/soft/trolcommander#:~:text=unsuccessful%20attempts%20to%20contact%20the%20authors)

**Comment by goyalyashpal** [comment]
hi @trol73 

can u think about this?

even if not, then can u atleast compile a list of the features added in trolcommander compared to kucommander.

regardless of ur decision, thanks

**Comment by trol73** [comment]
Hi,
trolCommander is open source software and you can move any features from it.
I personally prefer to develop trolCommander because i use it. And I use it, In particular because the muCommander seems to me to be too slow to start.

**Comment by goyalyashpal** [comment]
> is open source software and you can move any features from it.

yes, i perfectly understand that it is open source. but moving features require knowing which features exist in first place 😃 

so, if u can list down the feature additions in this one, then someone else can see if thats worth moving or not.

> And I use it, In particular because the muCommander seems to me to be too slow to start.

did u try [doublecmd] ?

[doublecmd]: https://github.com/doublecmd/doublecmd

**Comment by trol73** [comment]
Unfortunately, I do not have a ready list of implemented features. But I tried to describe them here: https://trolsoft.ru/en/taxonomy/term/50

Yesterday I installed and looked at the mucommander and I liked it. Over the past years, it has become much better and the community is actively working on it.

I also watched Double Commander and  I really liked the latest version too. I used to try to use it, but it was a very long time ago.
I see that this project is actively supported. At startup, it consumes about 100MB of memory, unlike the mucommander, which consumes 10 times more :) And doublecommander starts instantly, unlike the mucommander.
trolcommander initially consumed about 300 MB of memory on my computer. Now, on current versions of JVM, memory consumption has grown to several gigabytes, and this is terrible and unacceptable! :)

I think I'll try to use dcommander more actively. And, I hope, I will be able to fully migrate to it and participate in its development. :)

**Comment by goyalyashpal** [comment]
yep, doublecommander is super phenomenal - it's just sooo awesome. i have always wanted something like that. i have never every used windows explorer ever since i started using it. it was an instant click for me.



> I think I'll try to use dcommander more actively. And, I hope, I will be able to fully migrate to it and participate in its development. :)

that would be awesome :)
there were some small quality of life features from mucommander that doublecmd can use :)

like this one: https://github.com/doublecmd/doublecmd/issues/846

**Comment by goyalyashpal** [comment]
u might miss the inbuilt terminal though - i think

but u should try to use it anyway :)
